GitUI/CommandsDialogs/SettingsDialog/Pages/ColorsSettingsPage.cs

```diff
@@ -34,10 +35,17 @@ private ThemeId SelectedThemeId
             }
             set
             {
-                int index = _NO_TRANSLATE_cbSelectTheme.Items.IndexOf(new FormattedThemeId(value));
+                var formattedThemeId = new FormattedThemeId(value);
+                int index = _NO_TRANSLATE_cbSelectTheme.Items.IndexOf(formattedThemeId);
                 if (index < 0)
                 {
-                    throw new InvalidOperationException("Themes were not added to ComboBox");
+                    // Handle case when selected theme is missing gracefully.
+                    // It may happen in a following scenario:
+                    // - user creates custom theme and selects it in this settings page
+                    // - user saves app settings
+                    // - user deletes the file with custom theme
+                    Trace.WriteLine("Theme not found: " + formattedThemeId);
+                    index = 0;
```

Comment:
What happens if there are no themes? And the combo stays empty?

Reply:
If no themes combo still has 1 entry for default theme

Reply:
And the default theme can't be deleted, right?

Reply:
Right, there is no file for it )